subnet/subnet.go

```diff
@@ -44,26 +44,37 @@ type (
 	EventType int
 
 	Event struct {
-		Type  EventType `json:"type"`
-		Lease Lease     `json:"lease"`
+		Type    EventType `json:"type"`
+		Lease   Lease     `json:"lease"`
+		Network string    `json:"network"`
 	}
 )
 
 const (
 	SubnetAdded EventType = iota
 	SubnetRemoved
+	NetworkAdded
```

Comment:
Let's just have two events: `Added` and `Removed`. Or maybe call them `EventAdded` and `EventRemoved`. But they will be shared between both the lease and network events. There should be no confusion since the API endpoints and methods are different.